docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the code.
-FROM unocha/unified-builder:${TAG:-7.4-develop} as builder
+FROM public.ecr.aws/unocha/unified-builder:8.0-experimental as builder
```

Comment:
This flips it from the DocherHub registry to the AWS one. I think that's fine, though generally I'd like to swap them over once the *site* moves to AWS as well.

Reply:
@cafuego Do you mean having the *site* images on AWS?

Reply:
Yeah, we'll eventually stop using dockerhub. RW9 is already in the AWS registry and not on dockerhub.